packages/frontend-core/src/fetch/index.ts

```diff
@@ -1,19 +1,30 @@
-import TableFetch from "./TableFetch"
-import ViewFetch from "./ViewFetch"
-import ViewV2Fetch from "./ViewV2Fetch"
-import QueryFetch from "./QueryFetch"
-import RelationshipFetch from "./RelationshipFetch"
-import NestedProviderFetch from "./NestedProviderFetch"
-import FieldFetch from "./FieldFetch"
+import TableFetch, { TableDatasource } from "./TableFetch"
+import ViewFetch, { ViewV1Datasource } from "./ViewFetch"
+import ViewV2Fetch, { ViewDatasource } from "./ViewV2Fetch"
+import QueryFetch, { QueryDatasource } from "./QueryFetch"
+import RelationshipFetch, { RelationshipDatasource } from "./RelationshipFetch"
+import NestedProviderFetch, {
+  NestedProviderDatasource,
+} from "./NestedProviderFetch"
+import FieldFetch, { FieldDatasource } from "./FieldFetch"
 import JSONArrayFetch from "./JSONArrayFetch"
-import UserFetch from "./UserFetch"
-import GroupUserFetch from "./GroupUserFetch"
-import CustomFetch from "./CustomFetch"
+import UserFetch, { UserDatasource } from "./UserFetch"
```

Comment:
This was my previous point - its nice to just have these in types instead of needing to export like this.